fwoa-tools/src/migrationVerify.test.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+jest.mock('./migrationUtils', () => ({
+    checkConfiguration: () => {},
+    getFhirClientSMART: () => {
+        //eslint-disable-next-line
+        return axios.create();
+    },
+    getFhirClient: () => {
+        //eslint-disable-next-line
+        return axios.create();
+    }
+}));
+
+import AWS from 'aws-sdk';
+import {GetObjectRequest} from "aws-sdk/clients/s3";
```

Comment:
Nit: the formatting here looks off, should it be `import { GetObjectRequest } from "aws-sdk/clients/s3"`?